ts/src/xt.ts

```diff
@@ -112,7 +112,7 @@ export default class xt extends Exchange {
                 'transfer': true,
                 'withdraw': true,
             },
-            'precisionMode': DECIMAL_PLACES,
+            'precisionMode': TICK_SIZE,
```

Comment:
@Dan-krm do you agree with these changes?

Reply:
@Dan-krm I am almost sure that by the time you tested with TICK_SIZE precision, it couldnt work, because the "precision parsing" were not done till date there (see `parsePrecision`). 

you can take a look into this - https://github.com/orgs/ccxt/projects/28 we had migrated dozens of exchanges in the past and we should do it everywhere.

Reply:
@ttodua I'm not up to date on the precisions unification, so basically you're using parsePrecision to convert the exchanges precision from DECIMAL_PLACES to TICK_SIZE?

Reply:
yes, and I'll be making a pr to rename that method into `convertDigitPrecisionIntoTickSize`

Reply:
Okay I think these changes should be okay then